lib/cancel-stuck-txs.js

```diff
@@ -0,0 +1,171 @@
+import pRetry from 'p-retry'
+import assert from 'node:assert'
+import * as Sentry from '@sentry/node'
+import ms from 'ms'
+import timers from 'node:timers/promises'
+
+const ROUND_LENGTH_MS = ms('20 minutes')
+
+/**
+  * @param {string} f4addr
+  * @returns 1000 oldest messages
+  */
+async function getMessagesInMempool (f4addr) {
+  const res = await pRetry(
+    () => fetch(`https://filfox.info/api/v1/message/mempool/filtered-list?address=${f4addr}&pageSize=1000`),
```

Comment:
Should we uri-encode the address, just to be safe?
```suggestion
    () => fetch(`https://filfox.info/api/v1/message/mempool/filtered-list?address=${encodeURIComponent(f4addr)}&pageSize=1000`),
```

Reply:
I don't see why, the address comes from our ethers signer, which we're trusting everywhere else too